docs/wiki/Getting started - Get module cross-references.md

```diff
@@ -13,15 +13,20 @@ The `'Get-LinkedLocalModuleList'` function helps you with discovering cross-modu
 ---
 # Location
 
-You can find the script under `'utilities/tools/Get-LinkedLocalModuleList.ps1'`.
+You can find the script under `'utilities/tools/Get-LinkedModuleList.ps1'`.
 
 # How it works
 
 When invoking the script:
 
-1. The function leverages the utility [Get-LinkedModuleList](#related-function-get-linkedmodulelist) to fetch all references implemented in the modules in a given path.
-1. The function filters these references down to only local references (i.e., cross-module references) and formats them to show a consistent '\<ProviderNameSpace\>/\<ResourceType\>' format.
-1. Finally, it prints the references to the invoking terminal, group by ResourceType.
+1. The function fetches all references implemented in the modules in a given path.
+1. The function formats them to show a consistent '\<ProviderNameSpace\>/\<ResourceType\>' format.
+1. Finally, depending on whether you provided the `PrintLocalReferencesOnly` switch statement, the function either
```

Comment:
Do we have to have this via a switch? Why not write it to the host regardless and the returned output is what we use for the one for the readme?

Reply:
Don't really see the negative here. The good thing about this switch being disabled by default is the reduced vebosity that would otherwise pollute the pipeline log with likely unreadable content (if the wide).

Reply:
Wouldn't say its a negativity. Thought might be another way to not have an additional step the user has to do or think of. Maybe if we use format table to make it a prettier output, and I don't know how much it will pollute the pipeline given its usually 1 or 2 cross-referenced modules?

Reply:
The script actually returns all dependencies (ideally for all modules) and not just a single one. You can provide a path to a specific subfolder - but then it will also only find the dependencies within that subfolder.
I can also just remove the output if having it seems wrong. I figured it was useful - but would not provide us with any benefit in the pipeline. 
Note: This output (for all modules) would be produced for each Pester test run, as it runs the script as part of the Set-ModuleRedMe check.

Reply:
To avoid having the output impacting the pester test. Would it be better to decouple the functionality in a way that both the user script and the pester script benefits from it? Seems sharing a functionality that is intended to be user facing but at the same time use it for a CI function. Thoughts?

Reply:
I'm not sure I follow. I could however offer the following: I can remove the switch and instead just leave the output as a verbose message (removing the hardcoded verbose switch that is currently implemented). So if the user wants to see some output they could invoke the function with a `-Verbose` and if they want to mimic the exact behavior, invoke it like
`$null = Get-LinkedModuleList -Verbose`

Reply:
Yes that works too. Will explain what I meant later. Thanks @MrMCake !

Reply:
done

Reply:
thanks @MrMCake . I think you forgot to remove the switch from the wiki :D

Reply:
Damn :D